BitwardenShared/Core/Auth/Repositories/AuthRepository.swift

```diff
@@ -500,6 +505,10 @@ extension DefaultAuthRepository: AuthRepository {
     func existingAccountUserId(email: String) async -> String? {
         let matchingUserIds = await stateService.getUserIds(email: email)
         for userId in matchingUserIds {
+            // Skip unauthenticated user accounts, since the user may be trying to log back into an
+            // account that was soft logged out.
+            guard await (try? stateService.isAuthenticated(userId: userId)) == true else { continue }
```

Comment:
🤔 Is it ok to ignore the error here and not even log it?

Reply:
Ah, I can log it here if it throws.